src/templates/layouts/healthCareLocalFacility/OperatingStatus.tsx

```diff
@@ -1,5 +1,9 @@
 import { FacilityOperatingStatusFlags } from '@/types/drupal/node'
-import { HealthCareLocalFacility as FormattedHealthCareLocalFacility } from '@/types/formatted/healthCareLocalFacility'
+
+interface OperatingStatusFlagsProps {
+  operatingStatusFacility: FacilityOperatingStatusFlags
+  basePath: string
+}
```

Comment:
I changed this component because I'm going to start using it in my template. That probably means it should move out of this folder, but I didn't think we were ready for that. I actually had more code related to the usage of this, but I accidentally clobbered it beyond recovery, like a noob.